dist/steps/run_tests.sh

```diff
@@ -114,3 +114,19 @@ for platform in ${TEST_PLATFORMS//;/ }; do
     cat "$FULL_ARTIFACTS_PATH/$platform-results.xml" | grep test-run | grep Passed
   fi
 done
+
+#
+# Permissions
+#
+
+# Make a given user owner of all artifacts
+if [[ -n "$CHOWN_FILES_TO" ]]; then
+  chown -R "$CHOWN_FILES_TO" "$UNITY_PROJECT_PATH"
+  chown -R "$CHOWN_FILES_TO" "$FULL_ARTIFACTS_PATH"
+  chown -R "$CHOWN_FILES_TO" "$FULL_COVERAGE_RESULTS_PATH"
+fi
+
+# Add read permissions for everyone to all artifacts
+chmod -R a+r "$UNITY_PROJECT_PATH"
+chmod -R a+r "$FULL_ARTIFACTS_PATH"
+chmod -R a+r "$FULL_COVERAGE_RESULTS_PATH"
```

Comment:
Missing newline (recommend installing editorconfig plugin)

```suggestion
chmod -R a+r "$FULL_COVERAGE_RESULTS_PATH"

```

Reply:
Great, i installed the plugin and added the newline